src/test/java/com/ddudu/auth/service/AuthServiceTest.java

```diff
@@ -124,5 +129,46 @@ void setUp() {
       assertThat(actualId).isEqualTo(expected.getId());
       assertThat(actualAuthority).isEqualTo(expected.getAuthority());
     }
+
+  }
+
+  @Nested
+  class 사용자_로딩_테스트 {
+
+    @Test
+    void 존재하지_않는_사용자_아이디_로딩을_실패한다() {
+      // given
+      long randomId = faker.random()
```

Comment:
```suggestion
      long invalidId = faker.random()
```

저도 처음에 인재님 코드보고 `randomId`라고 적었는데, 존재하지 않는 사용자이기에 `invalidId`가 조금 더 맞지 않나 하고 invalidId로 바꾸었긴 합니다..! 사소한 걸지도 모르지만, 내일 전체 리팩토링할 때 이야기 해보면 좋을 것 같긴한데.. 이런 것까지 상세하게 통일하지 않아도 된다면 그대로 진행해도 됩니다!

Reply:
이거 저도 신경쓰였던 부분인데, 저는 invalidId로 쓰긴 하거든요! 
약간 invalid가 random을 포함한다하고 생각하기도 했는데, 이부분은 제가 리팩토링 노션 페이지에 넣어놓겠습니당!